tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: add addon to SLES via SCC
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base "y2logsstep";
+use strict;
+use testapi;
+use registration 'fill_in_registration_data';
+
+sub run() {
+    x11_start_program('xterm');
+    # add every used addon to regurl for proxy SCC
+    if (get_var('SCC_ADDONS')) {
+        my @addon_proxy = ("url: http://server-" . get_var('BUILD_SLE'));
+        for my $addon (split(/,/, get_var('SCC_ADDONS', ''))) {
+            my $uc_addon = uc $addon;    # change to uppercase to match variable
+            push(@addon_proxy, "\b.$addon-" . get_var("BUILD_$uc_addon"));
+        }
+        script_sudo "echo \"@addon_proxy.proxy.scc.suse.de\" > /etc/SUSEConnect", 0;
+    }
+    type_string "killall xterm\n";
+    x11_start_program("xdg-su -c '/sbin/yast2'");
```

Comment:
Running yast2 in root console is not same as via xdg-su, this has nothing with needles and it doesn't work without full path here